README.md

```diff
@@ -16,12 +16,16 @@ package main
 
 import (
 	"fmt"
+	"log"
+
 	"github.com/VojtechVitek/go-trello"
 )
 
 func main() {
 	// New Trello Client
-	trello, err := trello.NewAuthClient("application-key", "token")
+	appKey := "application-key"
+	token := "token"
+	trello, err := trello.NewAuthClient(appKey, &token)
```

Comment:
`UpdateToken()` may be difficult, since it relies on the web interface to 'allow' the token to be generated. 
I'm just starging in golang, so open to hearing options/suggestions.